model/host/db.go

```diff
@@ -78,6 +78,12 @@ var (
 	SpawnOptionsBuildIDKey       = bsonutil.MustHaveTag(SpawnOptions{}, "BuildID")
 	SpawnOptionsTimeoutKey       = bsonutil.MustHaveTag(SpawnOptions{}, "TimeoutTeardown")
 	SpawnOptionsSpawnedByTaskKey = bsonutil.MustHaveTag(SpawnOptions{}, "SpawnedByTask")
+	DistroIDKey                  = bsonutil.MustHaveTag(distro.Distro{}, "Id")
+)
+
+var (
+	HostsByDistroDistroIDKey = bsonutil.MustHaveTag(HostsByDistro{}, "DistroID")
```

Comment:
Unless you're using the struct elsewhere, I would define these as constants within the aggregation function